registry/storage/driver/base/base.go

```diff
@@ -42,7 +42,9 @@ import (
 	"io"
 
 	dcontext "github.com/docker/distribution/context"
+	"github.com/docker/distribution/metrics/prometheus"
 	storagedriver "github.com/docker/distribution/registry/storage/driver"
+	"time"
 )
 
```

Comment:
Make sure to decorate all the operations. Do we have a way to see writer duration? Writer bytes?